cdap-watchdog/src/main/java/co/cask/cdap/metrics/process/MessagingMetricsProcessorService.java

```diff
@@ -221,75 +278,72 @@ private void processMetrics() {
         MessageFetcher fetcher = messagingService.prepareFetch(topicIdMetaKey.getTopicId());
         fetcher.setLimit(fetcherLimit);
         if (lastMessageId != null) {
+          LOG.debug("Start fetching from lastMessageId = {}", lastMessageId);
```

Comment:
Log4j checks the Log-Level first before constructing the log message, so `if (LOG.isDebugEnabled())` may not be necessary?